doc/using_tf.rst

```diff
@@ -133,6 +133,26 @@ In your training script the channels will be stored in environment variables ``S
    ``output_path``.
 
 
+Use third-party libraries
+-------------------------
+
+If there are other packages you want to use with your script, you can use a ``requirements.txt`` to install other dependencies at runtime.
+
+Support for installing packages using ``requirements.txt`` varies with Tensorflow versions:
+
+- For Tensorflow 1.15.2 with Python 3.7 or newer, Tensorflow 2.2 or newer, and Script Mode without Horovod in any version:
```

Comment:
"any version" is a little ambiguous here - maybe change it to "TensorFlow 1.11 or newer"?